### ISSUE TITLE
Add Panozzo experiences

### ISSUE THREAD

**Issue by xavijam** [xavijam/mochiling#3]

**Comment by xavijam** [comment]
Add whatever

**Comment by esmochiling** [comment]
pais: Korea
texto: I'm pretty proud of this one, like I'm proud of my 10 years in this country.  Growing up in South Korea, that's what this is about.  I hope all PNU historians, past, present and future appreciate this.  I'm happy to hear these videos are inducing a ton of nostalgia.  In making this album, I thought often of how Korea changed me and, if and when I do leave, I'm happy I'll have this to remember Korea by.  Enjoy!
video: https://youtu.be/QNg1jcuu9ck?list=PLpQSL5pKUqhwUNzb_-GqyeRtok_-YBAPq  


Phillipines

- Texto: We will Love, We will Learn and we will be Happy!

- Video: https://youtu.be/f3P4yX4DweU

**Comment by xavijam** [comment]
Hecho y publicado

**Comment by xavijam** [comment]
Check Korea video for guessing date

**Comment by xavijam** [comment]
Falta el texto largo de Filipinas Vic

**Comment by xavijam** [comment]
http://mochiling.es/experiencias/daniel/filipinas/

**Comment by esmochiling** [comment]
esperando el texto introducción  de Dani... en cuanto lo mande te lo traduzco y te completo esto...

**Comment by esmochiling** [comment]
texto largo : 
 My wife and I went to Puerto Princessa, anticipating falling in love with El Nido, as most travelers do.  El Nido was impressive, but, in the truest Mochilling way, we discovered a hidden gem and left our hearts in Port Barton. Each night, the electricity is turned off at 7 to save power and you're treated to a sky full of stars and the smells of chicken sticks.  There are basketball games down each little dirt road and the people are just so happy to have you.  Port Barton, get it while it's paradise!

Mi mujer y yo fuimos a Puerto Princesa, pensando que nos enamoraríamos al instante de este lugar.
El Nido es increíble pero, de la forma mas mochiling, descubrimos esta gema escondida y dejamos nuestro corazones ahí...
Cada noche, el corte de electricidad de las 7 daba paso a un cielo lleno de estrellas y olor a pinchos de pollo cocinandose al fuego lento. Partidos de basket y charlas con locales en cada esquina. Port Barton, conócelo mientras siga siendo el paraíso.
